ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -0,0 +1,133 @@
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  intermediate-arm64:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  latest-arm64:
+    release:
+      architecture: arm64
+      channel: candidate
+      version: "4.12"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+  target-arm64:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-ovn-f14
+  cron: 4 4 3,18 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest-arm64
+      RELEASE_IMAGE_INTERMEDIATE: release:intermediate-arm64
+      RELEASE_IMAGE_TARGET: release:target-arm64
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_INTERMEDIATE_TAGS: '@arm64 and @upgrade-sanity and @4.13 and @aws-ipi
+        and not @fips'
+      E2E_RUN_LATEST_TAGS: '@arm64 and @upgrade-sanity and @4.14 and @aws-ipi and
+        not @fips'
+      OCP_ARCH: arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @upgrade-check and @4.14 and @aws-ipi and
+        not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@arm64 and @upgrade-prepare and @4.12 and @aws-ipi and
+        not @fips'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test-paused
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: azure-ipi-f14
+  cron: 26 18 10,25 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest-arm64
+      RELEASE_IMAGE_INTERMEDIATE: release:intermediate-arm64
+      RELEASE_IMAGE_TARGET: release:target-arm64
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      E2E_RUN_INTERMEDIATE_TAGS: '@arm64 and @4.13 and @azure-ipi and @upgrade-sanity
+        and not @fips'
+      E2E_RUN_LATEST_TAGS: '@arm64 and @4.14 and @azure-ipi and @upgrade-sanity and
+        not @fips'
+      OCP_ARCH: arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @4.14 and @azure-ipi and @upgrade-check
```

Comment:
sounds `@4.12` here?

Reply:
ah, makes sense. it should check 4.12 features working as expected. Thanks for the careful review